build.sbt

```diff
@@ -63,6 +63,8 @@ libraryDependencies ++= {
 
 Revolver.settings.settings
 
+Revolver.enableDebugging(port = 5051, suspend = false)
```

Comment:
This is awesome. I was wondering how to do this. Two things: (1) Will this cause debugging to be enabled when this is deployed to production, and (2) can you call this out in the README so developers are aware it exists?